src/auto-complete.jsx

```diff
@@ -38,11 +38,12 @@ const AutoComplete = React.createClass({
     onUpdateInput: React.PropTypes.func,
     open: React.PropTypes.bool,
     searchText: React.PropTypes.string,
-    showAllItems: React.PropTypes.bool,
+    //showAllItems is deprecated, use noFilter instead
```

Comment:
umm well indeed. Those two doesn't seem compatible. So I removed it.
updating the comment...